src/main/io/ledstrip.c

```diff
@@ -646,23 +646,25 @@ static void applyLedVtxLayer(bool updateNow, timeUs_t *timer)
     }
     else { // show frequency
         // calculate the VTX color based on frequency
+        int i = 0;
         if (frequency <= 5672) {
-            color = HSV(WHITE);
+            int i = 1;
```

Comment:
You are redeclaring `i` here. This won't work.
Also, if you use `colorIndex = COLOR_WHITE` the code will be a lot easier to read. (In general, avoid using 'magic numbers'.)